content/en/blog/2024/elastic-contributes-continuous-profiling-agent.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Elastic Contributes its Continuous Profiling Agent to OpenTelemetry
+linkTitle: Elastic Contributes Profiling Agent # Mandatory, make sure that your short title.
+date: 2024-06-06 # Put the current date, we will keep the date updated until your PR is merged
+# prettier-ignore
+cSpell:ignore: Bahubali Shetti Dmitry Filimonov Geisendörfer Halliday Christos Kalkanis
+author:
+  >- # If you have only one author, then add the single name on this line in quotes.
+  [Bahubali Shetti](https://github.com/bshetti) (Elastic), [Alexander
+  Wert](https://github.com/AlexanderWert) (Elastic), [Morgan
+  McLean](https://github.com/mtwo) (Splunk), [Ryan
+  Perry](https://github.com/Rperry2174) (Grafana)
+draft: true # TODO: remove this line once your post is ready to be published
+# canonical_url: http://somewhere.else/ # TODO: if this blog post has been posted somewhere else already, uncomment & provide the canonical URL here.
+issue: https://github.com/open-telemetry/community/issues/1918
+sig: Profiling SIG
+---
+
+Following significant collaboration between
+[Elastic](https://www.elastic.co/observability-labs/blog/elastic-donation-proposal-to-contribute-profiling-agent-to-opentelemetry)
+and [OpenTelemetry's profiling community](/blog/2024/profiling/), which included a
+thorough review process, we’re excited to announce that the OpenTelemetry
+project has accepted
+[Elastic's donation of its continuous profiling agent](https://github.com/open-telemetry/community/issues/1918).
+
+This marks a significant milestone in establishing profiling as a core telemetry
+signal in OpenTelemetry. Elastic’s [eBPF](https://ebpf.io/) profiling agent
+observes code across different programming languages and runtimes, third-party
+libraries, kernel operations, and system resources with low CPU and memory
+overhead in production. Both, SREs and developers can now benefit from these
+capabilities: quickly identifying performance bottlenecks, maximizing resource
+utilization, reducing carbon footprint, and optimizing cloud spend.
+
+Elastic’s decision to contribute the project to OpenTelemetry was made to
+accelerate OpenTelemetry’s mission and enable effective observability through
+high-quality, portable telemetry. This collaboration also shows the commitment
+to vendor neutrality and community-driven development enhancing the overall
+profiling and observability ecosystems.
+
+The Elastic team would like to thank the OpenTelemetry community for the great
+and constructive cooperation on the donation proposal. We look forward to
+jointly establishing continuous profiling as an integral part of OpenTelemetry.
+
```

Comment:
Since this is the otel blog, shouldn't it be the other way round, otel thanking elastic? Otherwise the blog post is written by elastic contributors.

Reply:
rephrased